cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -167,6 +170,42 @@ jobs:
           cat uploads.txt
           gh release upload ${{ github.ref_name }} $(cat uploads.txt)
           echo "uploaded!"
+          echo "releases=$(jq --compact-output ".releases" dist-manifest.json)" >> "$GITHUB_OUTPUT"
+
+{{%- if tap %}}
+  upload-homebrew-formula:
+    needs: upload-global-artifacts
+    runs-on: {{{ global_task.runner }}}
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      RELEASES: ${{ needs.upload-global-artifacts.outputs.releases }}
+      GITHUB_USER: "axo bot"
+      GITHUB_EMAIL: "admin+bot@axo.dev"
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: {{{ tap }}}
+          token: ${{ secrets.HOMEBREW_TAP_TOKEN }}
+      # So we have access to the formula
+      - name: Fetch local artifacts
+        run: |
+          gh release download ${{ github.ref_name }} --dir Formula --repo ${GITHUB_REPOSITORY} --clobber
+      - name: Commit formula files
+        run: |
+          git config --global user.name "${GITHUB_USER}"
+          git config --global user.email "${GITHUB_EMAIL}"
+
+          for release in $(echo $RELEASES | jq --compact-output '.[]'); do
+            name=$(echo $release | jq .app_name --raw-output)
+            version=$(echo $release | jq .app_version --raw-output)
+
+            git add Formula/${name}.rb
+            git commit -m "${name} ${version}"
```

Comment:
just confirming: tagging would be useless since every commit to this repo is effectively a tag, yeah?

Reply:
this is an interesting comment- while i agree every commit is effectively a tag- since our other tooling is using the github releases endpoint- is there a universe where we would want to consume info from the tap repo using that api? if so, it may be useful to tag?

Reply:
From a Homebrew perspective, there's no expectation of tagging within a tap. `brew update` just updates to the latest git tip, and there isn't a mechanism to use tags for anything.